src/Test.Utilities/CSharpCodeFixVerifier`2.cs

```diff
@@ -28,6 +28,7 @@ public static async Task VerifyAnalyzerAsync(string source, params DiagnosticRes
             var test = new Test
             {
                 TestCode = source,
+                MarkupOptions = MarkupOptions.UseFirstDescriptor
```

Comment:
What's the purpose of passing a `MarkupOptions` value here, where it would affect all unit tests?

Reply:
Without this i cannot use markup syntax for unit tests, e.g when analyzer has more than 1 descriptors this option need to set to tell which descriptor to use. Yes it will affect all tests but i though setting this to use the first descriptor by default wouldn't hurt and even useful so that you don't need to set all-time unless you want to use a specific descriptor. My plan was to see if anybody against setting this option like this, then i will handle it differently

Reply:
I am gonna remove this and add it into local helper, anyways tests also need  `ReferenceAssemblies.NetCore.NetCoreApp50`